projects/wave-core-new/src/lib/layers/symbology/symbology.model.ts

```diff
@@ -35,711 +49,733 @@ export const MIN_ALLOWED_POINT_RADIUS = 1;
 export const MAX_ALLOWED_POINT_RADIUS = 100;
 export const MAX_ALLOWED_TEXT_LENGTH = 25;
 
-/**
- * Serialization interface
- */
-export interface SymbologyDict {
-    symbologyType: string;
-}
+// export type StrokeDashStyle = Array<number>;
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface ISymbology {}
-
-export type StrokeDashStyle = Array<number>;
+const styleCache: {[key: string]: OlStyle} = {};
```

Comment:
maybe `STYLE_CACHE` since it is a static/constant?

Reply:
449a8a34ffcfa29f49afba5b97a1188706887ac4